### DISCUSSION TITLE
Langfuse in PHP. Which are the minimum functions needed to implement?

### DISCUSSION THREAD

**Discussion by phpia** [langfuse/langfuse#2007] [Support]
hey i want to test langfuse but my code is in php 
so which are the minimum functions from https://api.reference.langfuse.com/#post-/api/public/dataset-items that i need to implemnt? 
Thx

**Comment by marcklingen** [comment]
this really depends on which of the functionality that you want to use. for tracing that's mostly the /ingestion endpoint. for the other product features, you'd need to use their respective endpoints

**Comment by jesse-bos-dij** [comment]
You might want to check out [`langfuse-php`](https://github.com/DIJ-digital/langfuse-php), a PHP client for Langfuse. Could be useful if you’re looking to integrate Langfuse features like tracing and prompt management directly in your PHP project.
